pkg/mounter/oss/ossfs.go

```diff
@@ -301,6 +301,14 @@ func (o *Options) getAuthOptions(region string) (mountOptions []string) {
 }
 
 func (f *fuseOssfs) AddDefaultMountOptions(options []string) []string {
+	defaultOSSFSOptions := os.Getenv("DEFAULT_OSSFS_OPTIONS")
+	if defaultOSSFSOptions != "" {
+		optList := strings.Split(defaultOSSFSOptions, ",")
+		for _, opt := range optList {
+			options = append(options, strings.TrimSpace(opt))
```

Comment:
Are you 100% sure that space is not valid in an option? e.g., I think `passwd_file` can take a path ending with space? We should take extra care here because this is an API. It's behavior may not change theoretically.

Reply:
I don't know any, but i's just for private cloud now.  
Maybe @iawia002 can consider here, I think it depend on if this ENV is decided to configured by users.
I will not trimspace here in this PR.